Lib/sysconfig.py

```diff
@@ -112,6 +112,11 @@ def _safe_realpath(path):
 # PC/VS7.1
 if os.name == "nt" and "\\pc\\v" in _PROJECT_BASE[-10:].lower():
     _PROJECT_BASE = _safe_realpath(os.path.join(_PROJECT_BASE, pardir, pardir))
+# PC/VS9.0/amd64
+if (os.name == "nt" 
+   and os.path.basename(os.path.dirname(os.path.dirname(_PROJECT_BASE))).lower() == "pc" 
+   and os.path.basename(os.path.dirname(_PROJECT_BASE)).lower().startswith("vs")):
```

Comment:
Why not indent it under the previous check and just do `"\\pc\\vs9.0" in _PROJECT_BASE[-10:].lower()` (double check that, but I'm assuming the previous condition is okay).